furms-web-ui/src/main/java/io/imunity/furms/ui/views/site/settings/SettingsView.java

```diff
@@ -169,6 +172,12 @@ private void doSaveAction(Binder<SiteSettingsDto> binder) {
 						.build());
 				refreshBinder(binder);
 				showSuccessNotification(getTranslation("view.site-admin.settings.form.button.save.success"));
+			} catch (DuplicatedNameValidationError e) {
+				findFormField(binder, TextField.class, NAME_FIELD_ID)
```

Comment:
Why to search for a name field if you know that you want the name field? that's makes no sense :-) just use name.